custom_components/lifx/switch.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+from homeassistant.components.switch import ( SwitchEntity, SwitchEntityDescription, SwitchDeviceClass )
+
+from typing import Any
+from datetime import datetime, timedelta
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import CALLBACK_TYPE, HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.event import async_track_point_in_utc_time
+from homeassistant import util
+
+from .const import ( DOMAIN )
+from .coordinator import LIFXUpdateCoordinator
+from .entity import LIFXEntity
+from .util import lifx_features
+
+LIFX_SWITCH = SwitchEntityDescription(key="LIFX_SWITCH", name="LIFX Switch", device_class=SwitchDeviceClass.SWITCH)
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up LIFX from a config entry."""
+    domain_data = hass.data[DOMAIN]
+    coordinator: LIFXUpdateCoordinator = domain_data[entry.entry_id]
+    if lifx_features(coordinator.device)["relays"]:
+        async_add_entities([LIFXSwitch(coordinator, LIFX_SWITCH, 0), LIFXSwitch(coordinator, LIFX_SWITCH, 1), LIFXSwitch(coordinator, LIFX_SWITCH, 2), LIFXSwitch(coordinator, LIFX_SWITCH, 3)])
```

Comment:
What about 2-gang switches? Unfortunately, there's no way to tell from the API, so you may have to create a config flow for the switch.